conduce/util.py

```diff
@@ -497,16 +497,20 @@ def entities_to_entity_set(entity_list):
     return {'entities': entities}
 
 
+def parse_sample(sample):
+    if 'time' in sample:
+        sample['time'] = parser.parse(sample['time'])
+    return sample
+
+
 def parse_samples(samples):
     for sample in samples:
-        if 'time' in sample:
-            sample['time'] = parser.parse(sample['time'])
+        sample.update(parse_sample(sample))
     return samples
 
 
 def json_to_samples(json_path):
-    with open(json_path, 'r') as json_file:
-        return json.load(parse_samples, open(json_file))
```

Comment:
I totally fubar'd this and forgot I had unit tests for it.  One of the reasons it will be good to have automated testing on this repo.